tools/testsys/src/delete.rs

```diff
@@ -0,0 +1,27 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use futures::TryStreamExt;
+use model::test_manager::{DeleteEvent, TestManager};
+
+/// Delete all tests and resources from a testsys cluster.
+#[derive(Debug, Parser)]
+pub(crate) struct Delete {}
+
+impl Delete {
+    pub(crate) async fn run(self, client: TestManager) -> Result<()> {
+        let mut stream = client.delete_all().await.context("Unable to delete all")?;
+
+        while let Some(delete) = stream
+            .try_next()
+            .await
+            .context("A deletion error occured")?
+        {
+            match delete {
+                DeleteEvent::Starting(crd) => println!("Starting delete for {}", crd.name()),
+                DeleteEvent::Deleted(crd) => println!("Delete finished for {}", crd.name()),
+                DeleteEvent::Failed(crd) => println!("Delete failed for {}", crd.name()),
+            }
```

Comment:
not a blocker - consider doing what pubsys does and setting up a logger vs. `println` and `eprintln`.

Reply:
> not a blocker - consider doing what pubsys does and setting up a logger vs. println and eprintln.

I had also commented on this. @ecpullen has added a logger, but these print statements are part of the UI, (i.e. they are not logs) so I requested that they be changed back to println. We will use the logger for things that are more debugging/logging related, e.g. warnings and debug!, trace! statements.